docs/userguide/expectations.md

```diff
@@ -432,6 +433,243 @@ end;
 The `a_nulls_are_equal` parameter controls the behavior of a `null = null` comparison.
 To change the behavior of `NULL = NULL` comparison pass the `a_nulls_are_equal => false` to the `equal` matcher.  
 
+## include / contain
+
+This matcher supports only cursor comparison. It check if the give set contain all values from given subset.
+
+Test using this matcher behaves similar to `equal`  in respect that it succeeds only when the compared data-types are exactly the same.
+
+The matcher supports all advanced comparison options as `equal` e.g. include , exclude, join_by.
+
+The matcher will be successful only when all of the values in expected results are part of actual set.
+
+In situation where the duplicate is present in expected set we would also expect matching number of occurrences in actual set for matcher to be success.
+
+*Example 1*
+
+```sql
+   PROCEDURE ut_refcursors IS
+    l_actual   SYS_REFCURSOR;
+    l_expected SYS_REFCURSOR;
+  begin
+    --Arrange
+    open l_actual  for select rownum as rn  from dual a connect by level < 10;
+    open l_expected for select rownum as rn from dual a connect by level < 4
+    union all select rownum as rn from dual a connect by level < 4;
+    
+    --Act
+    ut.expect(l_actual).to_include(l_expected);
+   END;
+```
+
+Will result in failure message
+
+```sql
+  1) ut_refcursors
+      Actual: refcursor [ count = 9 ] was expected to include: refcursor [ count = 6 ]
+      Diff:
+      Rows: [ 3 differences ]
+      Missing:  <ROW><RN>3</RN></ROW>
+      Missing:  <ROW><RN>2</RN></ROW>
+      Missing:  <ROW><RN>1</RN></ROW>
+```
+
+
+
+Similar negated `not_to_include`/ `not_to_contain` will be successful only when none of the values from expected set are part of actual e.g.
```

Comment:
The negated version of `include / contain` ( `not_to_include / not_to_contain`) is successful only when all values from expected are not part of actual.

*Note: Maybe a set diagram would help?*
https://www.google.ie/search?q=venn+diagram+separate+sets&safe=off&source=lnms&tbm=isch&sa=X&ved=0ahUKEwiG-5OgoY3fAhUDgVwKHUMqDx0Q_AUIDigB&biw=1920&bih=969